test/test-applications/integrations/TestApplication.AspNet.NetFramework/TestApplication.AspNet.NetFramework.csproj

```diff
@@ -37,10 +37,16 @@
     <Reference Include="Microsoft.Web.Infrastructure">
       <HintPath>..\..\..\..\packages\Microsoft.Web.Infrastructure.2.0.0\lib\net40\Microsoft.Web.Infrastructure.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
I do not see any reference to Newtonsoft.Json? Is it really needed?
Can it be replaced by System.Text.Json?

Newtonsoft.Json has a lot of security issues last year.

Reply:
It came with WebApi package.